instrumentation/net/http/splunkhttp/handler.go

```diff
@@ -0,0 +1,89 @@
+// Copyright Splunk Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package splunkhttp
+
+import (
+	"net/http"
+	"os"
+	"strings"
+
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+)
+
+const (
+	EnvVarServerTimingEnabled = "SPLUNK_CONTEXT_SERVER_TIMING_ENABLED"
```

Comment:
I thought it may be good to export. Env vars are kind of public API anyway. Someone may like to use it 😉 I should probably add a comment anyway.

Reply:
Added comments https://github.com/signalfx/splunk-otel-go/pull/30/commits/33dec16e4a16090a3f0b13b567f4f30593df4daf

Reply:
I'm pretty hesitant to export this at this point. It will set a precedent and I'm not sure it is one we will or won't want in the long term.

I'm not aware of a situation where a user will need to reference this variable given it is expected to describe something set in the runtime environment prior to execution. But, if we do not export this now and there arises a need the future it will be simple enough to export it. In that situation we will probably want to look holistically at exporting all environment variables Splunk uses, likely in some kind of "splunk environment variable" package. If we export this now it will either require us to duplicate the declaration in both packages or be a breaking change.

Reply:
Unexported https://github.com/signalfx/splunk-otel-go/pull/30/commits/165c57c14f22bd11afdca17c60347dcc4b9c7208